dockerfile_scripts/scrape_libs.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/bash
 
+WHOAMI=$(whoami)
 # Only scrape host libs if AWS/NCCL/OFI was built for this image
 if [ -d /container/aws/lib ]
 then
    # See if we mounted in host libs in the expected location
-   host_dir="/host"
+   host_dir="/det_host"
```

Comment:
not sure I understand the difference in /det_host and /det_libfabric

Reply:
/det_libfabric is needed to prevent multiple libfabrics from being found and breaking the scrape.